renku/ui/cli/config.py

```diff
@@ -93,6 +93,10 @@
 | ``dataverse.server_url``       | URL for the Dataverse API server    | ``None``  |
 |                                | to use                              |           |
 +--------------------------------+-------------------------------------+-----------+
+| ``default_dataset_add_action`` | Default action when adding files to | ``None``  |
```

Comment:
Shouldn't the default be `copy` ?

Reply:
We'll prompt the user if nothing is configured here; to me, that's a default value of `None`. If the default value is `copy` then we shouldn't prompt when nothing is configured.